code/modules/clothing/ears/_ears.dm

```diff
@@ -22,6 +22,18 @@
 	. = ..()
 	AddElement(/datum/element/earhealing)
 
+/obj/item/clothing/ears/earmuffs/attackby(obj/item/I, mob/living/user) //NSV13 comment
+	if(istype(I,/obj/item/radio/headset))
+		var/obj/item/radio/headset/radiomuff/newmuf = new /obj/item/radio/headset/radiomuff(get_turf(user.loc))
+		newmuf.keyslot = I
+		newmuf.recalculateChannels()
+		to_chat(user,"<span class='notice'>You stuff the [I] into the [src].</span>")
+		qdel(I)
```

Comment:
uh, If I'm reading this right, you're setting the keyslot to the radio and then deleting it???

Get the keyslots of the radio itself.

Reply:
you also need to copy over the command and use_command values for loudmode as well.

Reply:
When testing it did allow me to access the radio frequencies, iam not entirely sure how/why it works either

Reply:
Should be fixed now? I tested loudmode and it worked. It should also copy the key over now